webpack.config.js

```diff
@@ -1,32 +1,132 @@
-module.exports = {
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+var precss = require('precss');
+var path = require('path');
+
+var exp = {
+  devServer: {
+    host: '0.0.0.0',
+    port: 8090
+  },
+  devtool: 'eval',
   entry: [
-    './node_modules/wouso-foundation/components/wouso-qotd/main.jsx',
-    './node_modules/wouso-foundation/components/wouso-quest/main.jsx',
-    './node_modules/wouso-foundation/components/messages.jsx',
-    './node_modules/wouso-foundation/components/profile.jsx',
-    './node_modules/wouso-foundation/components/admin.jsx'
+    './modules/wouso-foundation/components/wouso-qotd/main.jsx',
+    './modules/wouso-foundation/components/wouso-quest/main.jsx',
+    './modules/wouso-foundation/components/messages.jsx',
+    './modules/wouso-foundation/components/profile.jsx',
+    './modules/wouso-foundation/components/admin.jsx'
   ],
   output: {
-    filename: './node_modules/wouso-foundation/js/bundle.js'
+    path: path.join(__dirname, 'assets'),
+    filename: '[name].js',
+    publicPath: '/assets/'
   },
   module: {
-    loaders: [{
-      //tell webpack to use jsx-loader for all *.jsx files
-      test: /\.jsx$/,
-      loader: 'babel-loader',
-      exclude: /node_modules/,
-      query: {
-        presets:['react']
+    rules: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              plugins: [
+                'transform-class-properties',
+                'syntax-async-functions'
+              ],
+              presets: [
+                ['es2017'],
```

Comment:
Remove brackets and replace with "babel-preset-es2017"
Do `npm install babel-preset-es2017`